src/components/editor/faultTree/Editor.tsx

```diff
@@ -52,8 +52,7 @@ const Editor = ({ setAppBarName }: DashboardTitleProps) => {
       setAppBarName(faultTree.name);
       setRootEvent(faultTree.manifestingEvent);
       if (faultTree.faultEventScenarios) {
-        const result = getScenariosWithHighestProbability(faultTree.faultEventScenarios);
-        setFaultEventScenarios(result);
+        setFaultEventScenarios(getScenarioWithHighestProbability(faultTree.faultEventScenarios));
```

Comment:
If you want to keep this method, i would expect:
setFaultEventScenarios([getScenarioWithHighestProbability(faultTree.faultEventScenarios]));

Reply:
Changed it as u show